airbyte-integrations/connectors/source-typeform/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY main.py ./
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.12
```

Comment:
I believe this should be a minor version bump, not patch

Reply:
..or even major, because of backwards incompatible changes in the spec

Reply:
I agree, made a major version bump considering that changes are breaking